common/telemetry.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * This file is part of Bott.
+ *
+ * This project is dual-licensed:
+ * - Non-commercial use: AGPLv3 (see LICENSE file for full text).
+ * - Commercial use: Proprietary License (contact D@nielLaCos.se for details).
+ *
+ * Copyright (C) 2025 DanielLaCos.se
+ */
+
+import { type Span, trace, type Tracer } from "@opentelemetry/api";
+
+const TRACER_NAME = "bott";
+
+export function getTracer(): Tracer {
```

Comment:
for now let's just use one tracer for the whole project. no need for "getTracer"